ADAL/src/telemetry/ADTelemetryAPIEvent.m

```diff
@@ -27,6 +27,7 @@
 #import "ADTelemetryEventStrings.h"
 #import "ADHelpers.h"
 #import "ADAL_Internal.h"
+#import "ADAuthenticationError.h"
```

Comment:
nit: why this was added?

Reply:
I think it was added when I was experimenting with the code and then I forgot to removed it. Will remove it.